### PR TITLE
Move baseframe templates into funnel

### PR DESCRIPTION
**Pull request by vidya-ram**

### CODE REVIEW

File: funnel/templates/screens_layout.html.jinja2 [hunk 1]
```diff
@@ -5,18 +5,13 @@
 	{% assets 'css_screens' -%}
 	   <link rel="stylesheet" type="text/css" href="{{ ASSET_URL }}" />
 	{%- endassets %}
```
Comment (jace) [inline funnel/templates/screens_layout.html.jinja2:7]: Mixed tabs and spaces in this template too.

File: funnel/templates/screens_layout.html.jinja2 [hunk 2]
```diff
@@ -5,18 +5,13 @@
...
 {% block footer %}
-    {% block footer_logo %}
-    <div id="footerlogo">
-      {%- block footer_logo_image %}
-        <img src="{{ url_for('baseframe.static', filename='img/logo@2x.png') }}">
-      {%- endblock %}
-    </div>
-    {% endblock %}
+  {% block footer_logo %}
+  {% endblock %}
```
Comment (jace) [inline funnel/templates/screens_layout.html.jinja2:16]: What page uses footer logos?

File: funnel/templates/update_details.html.jinja2 [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {% set current_page = 'update' %}
```
Comment (jace) [inline funnel/templates/update_details.html.jinja2:1]: For a future PR: these `set` tags have to be removed, and the template should use `current_view.<whatever>`. I will refactor the views to reflect the page layouts and sections.

**Review by jace — CHANGES_REQUESTED** [review]
The layout template from baseframe needs some cleanup. Other small issues and comments.

File: funnel/templates/layout.html.jinja2 [hunk 4]
```diff
@@ -16,215 +30,310 @@
...
-            {% endblock %}
-          </div>
-        </div>
-      </div>
-    </div>
+  {%- block typekit %}
   {%- endblock %}
-{% endblock %}
 
+  {%- block layoutheaders %}
```
[the remark this answers is not in the document]
Comment (vidya-ram) [reply funnel/templates/layout.html.jinja2:80]: We use it the formlayout template

File: funnel/templates/update_details.html.jinja2 (hunk 3 above)
Comment (vidya-ram) [reply funnel/templates/update_details.html.jinja2:1]: Added an issue for this

File: funnel/templates/layout.html.jinja2 [hunk 5]
```diff
@@ -16,215 +30,310 @@
...
+    {% block jquery -%}
+      <script src="//ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
+      <script type="text/javascript">
+        if (typeof jQuery === 'undefined') {
+          var jqueryScript = document.createElement('script');
+          jqueryScript.setAttribute('type','text/javascript');
+          jqueryScript.setAttribute('src','{{ url_for("baseframe.static", filename="js/jquery-2.2.4.min.js") }}');
+          document.head.appendChild(jqueryScript);
+        }
+      </script>
```
[the remark this answers is not in the document]
Comment (vidya-ram) [reply funnel/templates/layout.html.jinja2:251]: Added an issue for this

File: funnel/templates/layout.html.jinja2 [hunk 6]
```diff
@@ -16,215 +30,310 @@
...
+    {%- endblock %}
+  <!--<![endif]-->
+  {%- for asset in config.get('ext_js', []) %}
+    <script type="text/javascript" src="{{ asset|ext_asset_url }}"></script>
+  {%- endfor %}
+  {#- This block is to include JS assets of the app that are not required on all the pages but has to be included before baseframe bundle(assets "js_all").
+  For instance codemirror-markdown-js.js is only required in the formlayout pages. CodeMirror functions are called from baseframe.js, hence it has to be included before  baseframe.js #}
+  {% block pagescripts %}{% endblock %}
```
[the remark this answers is not in the document]
Comment (vidya-ram) [reply funnel/templates/layout.html.jinja2:259]: baseframe.js has codemirror calls so this block is to include codemirror.js before it.

File: funnel/templates/layout.html.jinja2 [hunk 7]
```diff
@@ -16,215 +30,310 @@
...
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+      ga('create', {{ config['GA_CODE']|tojson }}, 'auto');
+      // Track Device Pixel Ratio
+      var pixelRatio = window.getDevicePixelRatio();
+      ga('set', 'metric1', pixelRatio);
+      ga('set', 'dimension1', {{ request.endpoint|tojson }});
+      ga('send', 'pageview');
+    </script>
```
[the remark this answers is not in the document]
Comment (vidya-ram) [reply funnel/templates/layout.html.jinja2:293]: Added an issue for this

File: funnel/static/js/libs/mui.js [hunk 8]
```diff
@@ -0,0 +1,1099 @@
+// "version": "0.9.21"
```
Comment (jace) [inline funnel/static/js/libs/mui.js:1]: This is not minimized. Prettier will automatically unminimize this as part of pre-commit, so it has to be added in exclusions. Ideally this comes from webpack.

File: funnel/templates/screens_layout.html.jinja2 [hunk 9]
```diff
@@ -3,20 +3,13 @@
 
 {% block pageheaders %}
 	{% assets 'css_screens' -%}
-	   <link rel="stylesheet" type="text/css" href="{{ ASSET_URL }}" />
+    <link rel="stylesheet" type="text/css" href="{{ ASSET_URL }}" />
 	{%- endassets %}
```
Comment (jace) [inline funnel/templates/screens_layout.html.jinja2:7]: This has tabs.

**Review by jace — CHANGES_REQUESTED** [review]
Two minor issues:

1. `mui.js` is not minimized. It should be renamed to `mui.min.js` and prettier has to be told to ignore `*.min.js` files.
2. One template has tabs instead of spaces.

**Pull request merged**